slack/web/client.py

```diff
@@ -2154,3 +2154,66 @@ def views_publish(
         else:
             kwargs.update({"view": view})
         return self.api_call("views.publish", json=kwargs)
+
+    def workflows_stepCompleted(
+        self,
+        *,
+        workflow_step_execute_id: str,
+        outputs: dict = {},
+        **kwargs
+    ) -> Union[Future, SlackResponse]:
+        """Indicate a successful outcome of a workflow step's execution.
+        Args:
+            workflow_step_execute_id (str): A unique identifier of the workflow step to be updated.
+                e.g. 'add_task'
+            outputs (dict): A key-value object of outputs from your step.
+                e.g. { 'task_name': 'Task Name' }
+        """
+        kwargs.update({"workflow_step_execute_id": workflow_step_execute_id})
+        if outputs:
+            kwargs.update({"outputs": outputs})
+
+        return self.api_call("workflows.stepCompleted", json=kwargs)
+
+    def workflows_stepFailed(
+        self,
+        *,
+        workflow_step_execute_id: str,
+        error: dict,
+        **kwargs
+    ) -> Union[Future, SlackResponse]:
+        """Indicate an unsuccessful outcome of a workflow step's execution.
+        Args:
+            workflow_step_execute_id (str): A unique identifier of the workflow step to be updated.
+                e.g. 'add_task'
+            error (dict): A dict with a message property that contains a human readable error message
+                e.g. { message: 'Step failed to execute.' }
+        """
+        kwargs.update({"workflow_step_execute_id": workflow_step_execute_id, "error": error})
+        return self.api_call("workflows.stepFailed", json=kwargs)
+
+    def workflows_updateStep(
+        self,
+        *,
+        workflow_step_edit_id: str,
+        inputs: dict = {},
```

Comment:
For other function signatures in this file, the arguments that are missing have tended to default to `None`. Since both empty `dict` and `list` are falsy, I fell back to those, but I'm happy to also default to `None`.